src/thread.ts

```diff
@@ -28,6 +28,10 @@ export interface Thread {
 export type SubprocessMessage = SubprocessOutputMessage | SubprocessRunnerMessage;
 
 export function supportsWorkerThreads(): boolean {
+  const useProcessesEnvVar = (process.env.USE_PROCESSES || '').toLowerCase();
+  if (['true', '1', 'yes', 'y'].indexOf(useProcessesEnvVar) > -1) {
+    return false;
+  }
```

Comment:
Let's call it `USE_CHILD_PROCESS` please. Also, it's usually better to have one option rather than a couple. So let's just stick with a `process.env.USE_CHILD_PROCESS === '1'` check.